src/backend/BSConfig.pm.template

```diff
@@ -155,8 +154,11 @@ our $relsync_pool = {
 #our $sign = '/usr/bin/sign';
 #Extend sign call with project name as argument "--project $NAME"
 #our $sign_project = 1;
+#Create a key by default for new projects, if top level project don't have one
+#our $forceprojectkeys = 1;
```

Comment:
This shouldn't be part of the template, it comes via configuration.xml meanwhile and I dropped it here on purpose.